modules/light-clients/09-localhost/keys.go

```diff
@@ -0,0 +1,9 @@
+package localhost
+
+const (
+	// ModuleName defines the 09-localhost light client module name
+	ModuleName = "09-localhost"
+)
+
+// SentinelProof defines the 09-localhost sentintel proof
+var SentinelProof = []byte("proof_localhost")
```

Comment:
I think we should explain a little of what `sentinel proof` means?

Reply:
I am not sure, but I think I remember @AdityaSripal saying that we could just use a slice with one byte?

Reply:
Updated